packages/charts/examples/components/demo/advanced-usage/events/event-sampler/event-sampler.component.html

```diff
@@ -13,14 +13,13 @@
                                [seriesRenderState]="chartAssist.renderStatesIndex[legendSeries.id]?.state"
                                [isSelected]="!chartAssist.isSeriesHidden(legendSeries.id)"
                                (isSelectedChange)="chartAssist.toggleSeries(legendSeries.id, $event)">
-                <nui-rich-legend-tile [value]="chartAssist.getHighlightedValue(legendSeries, 'y')"
-                                      [backgroundColor]="chartAssist.palette.standardColors.get(legendSeries.id)"
+                <nui-basic-legend-tile [value]="chartAssist.getHighlightedValue(legendSeries, 'y')"
                                       [color]="chartAssist.palette.textColors.get(legendSeries.id)">
                     <nui-chart-marker rightEdge
                                       [marker]="chartAssist.markers.get(legendSeries.id)"
                                       [color]="chartAssist.palette.standardColors.get(legendSeries.id)">
```

Comment:
the basic tile component doesn't use the rightEdge attribute, so this can be removed:

```suggestion
                    <nui-chart-marker [marker]="chartAssist.markers.get(legendSeries.id)"
                                                  [color]="chartAssist.palette.standardColors.get(legendSeries.id)">
```